uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Service/SettingsManager.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+interface CRM_HRLeaveAndAbsences_Service_SettingsManager {
+  public function get($setting);
```

Comment:
missing doc blocks and we can probably add a brief description of what the interface does.
Maybe If the doc blocks are defined here, the other child classes implementing it can inherit that.